Modules/TPC/include/TPC/ClustersData.h

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   Clusters.h
+/// \author Jens Wiechula
+/// \author Thomas Klemenz
+///
+
+#ifndef QC_MODULE_TPC_CLUSTERSDATA_H
+#define QC_MODULE_TPC_CLUSTERSDATA_H
+
+// ROOT includes
+#include "TObject.h"
+
+// O2 includes
+#include <Mergers/MergeInterface.h>
+#include <Rtypes.h>
+#include "TPCQC/Clusters.h"
+
+using o2::mergers::MergeInterface;
+using o2::tpc::qc::Clusters;
+
+namespace o2::quality_control_modules::tpc
+{
+
+class ClustersData final : public TObject, public MergeInterface
+{
+ public:
+  ClustersData()
+    : o2::mergers::MergeInterface(), TObject()
+  {
+  }
```

Comment:
Perhaps the constructor should take the name as argument, which defaults to `ClusterData`. If we want to use it for the Digits as well it should have a different name.

Reply:
I was actually thinking to do a separate class for the RawDigits task but I can try to do it within this one.

Reply:
I implemented a setName function that is called in the initialize function of the Clusters/RawDigits task to set the proper name of the data object.